### PR TITLE
Update to GNOME 3.30 runtime

### PR DESCRIPTION
**Pull request by andreasn**

### CODE REVIEW

**Comment by andreasn** [comment]
I realized since it was more or less identical to GNOME Books, I might as well do this one as well.

File: org.gnome.Documents.json [hunk 1]
```diff
@@ -105,50 +106,40 @@
             "sources": [
                 {
                     "type": "archive",
-                    "url": "https://poppler.freedesktop.org/poppler-data-0.4.9.tar.gz",
+                    "url": "https://poppler.freedesktop.org/poppler-data-0.4.9.tar.gz	",
```
Comment (TingPing) [inline org.gnome.Documents.json]: space at end.

**Comment by nedrichards** [comment]
bot, build

**Comment by nedrichards** [comment]
Build failed: https://flathub.org/builds/#/builders/2/builds/5962

**Comment by barthalion** [comment]
bot, build

**Pull request merged**